src/applications/facility-locator/actions/index.js

```diff
@@ -323,74 +323,63 @@ export const genBBoxFromAddress = query => {
  * from the coordinates center of the map
  */
 export const genSearchAreaFromCenter = query => {
-  const { lat, lng } = query;
+  const { lat, lng, currentMapBoundsDistance, currentBounds } = query;
   return dispatch => {
-    const types = TypeList;
-    mbxClient
-      .reverseGeocode({
-        countries: CountriesList,
-        types,
-        query: [lng, lat],
-      })
-      .send()
-      .then(({ body: { features } }) => {
-        const coordinates = features[0].center;
-        const zip =
-          features[0].context.find(v => v.id.includes('postcode')) || {};
-        const location = zip.text || features[0].place_name;
-        const featureBox = features[0].box;
-
-        const sw = coordinates[0] - BOUNDING_RADIUS;
-        const se = coordinates[1] - BOUNDING_RADIUS;
-        const nw = coordinates[0] + BOUNDING_RADIUS;
-        const ne = coordinates[1] + BOUNDING_RADIUS;
-        let minBounds = [sw, se, nw, ne];
-        if (featureBox) {
-          minBounds = [
-            Math.min(featureBox[0], sw),
-            Math.min(featureBox[1], se),
-            Math.max(featureBox[2], nw),
-            Math.max(featureBox[3], ne),
-          ];
-        }
+    if (currentMapBoundsDistance > 500) {
+      dispatch({ type: GEOCODE_FAILED });
+      dispatch({ type: SEARCH_FAILED, error: { type: 'mapBox' } });
+    } else {
+      const types = TypeList;
+      mbxClient
+        .reverseGeocode({
+          countries: CountriesList,
+          types,
+          query: [lng, lat],
+        })
+        .send()
+        .then(({ body: { features } }) => {
+          const zip =
+            features[0].context.find(v => v.id.includes('postcode')) || {};
+          const location = zip.text || features[0].place_name;
 
-        const radius = distBetween(
-          features[0].bbox[1],
-          features[0].bbox[0],
-          features[0].bbox[3],
-          features[0].bbox[2],
-        );
+          const radius = distBetween(
+            currentBounds[1],
+            currentBounds[0],
+            currentBounds[3],
+            currentBounds[2],
+          );
```

Comment:
Since we are not using "fitToBounds" when the map is panned/zoomed, I suggest we apply a small reduction in radius here to make sure all of the markers are clearly visible.

```suggestion
          const RADIUS_FUDGE_FACTOR = .95; // this is a total guess
          
          const radius = distBetween(
            currentBounds[1],
            currentBounds[0],
            currentBounds[3],
            currentBounds[2],
          ) * RADIUS_FUDGE_FACTOR;
```

Reply:
the issue was rounding the values, I think this fixes the exact bound - https://github.com/department-of-veterans-affairs/vets-website/pull/15126/commits/a80338e68b1eb37fe5fe94f25360b2c4f984891e